src/theming/sap_fiori_3.scss

```diff
@@ -1,3 +1,19 @@
 :root {
   --sapInverted_Object_Border_Width: 0;
+
+  /* Rating Indicator */
+  // default mode
+  --sapRating_Indicator_Selected_Icon_Color: #d08014;
```

Comment:
instead of hardcoding the values, use the parameter name `--sapContent_RatedColor`

Reply:
fixed